keyvi/src/cpp/dictionary/dictionary_compiler.h

```diff
@@ -160,14 +160,14 @@ class DictionaryCompiler
       value_store_->CloseFeeding();
       sorter_.end();
       sorter_.merge_runs();
+      CreateGenerator();
 
       // check that at least 1 item is there
       if (!sorter_.can_pull()) {
```

Comment:
What about if we remove this check and add `if ( sorter_.can_pull())` on line: 170 ?